Automattic/wp-calypso
# VideoPress Onboarding V2: Intro View

#### Proposed Changes

* Adds styles and a few tweaks to some props to get "powered by VideoPress" displaying at the bottom of the intro view.

<img width="1139" alt="Screen Shot 2022-09-14 at 2 37 52 PM" src="https://user-images.githubusercontent.com/789137/190257630-cb861573-16f3-4f78-a4c1-8b7dfc677c85.png">
<img width="193" alt="Screen Shot 2022-09-14 at 2 38 21 PM" src="https://user-images.githubusercontent.com/789137/190257642-99a574cf-9a41-4738-8996-28a82eb45e0d.png">

#### Testing Instructions

* Load up `setup/intro?flow=videopress`.
* Verify styles match the (i2) design.
* Try different screen sizes.
* Note: the button doesn't take you to the next step but I think we haven't set that up yet?
* Try a different flow like `setup/intro?flow=newsletter` and verify it still has `Jetpack powered` in the footer area.

#### Pre-merge Checklist




Related to pxWta-1z5-p2
